src/sonic_ax_impl/mibs/__init__.py

```diff
@@ -212,6 +212,7 @@ def config(**kwargs):
     global redis_kwargs
     redis_kwargs = {k:v for (k,v) in kwargs.items() if k in ['unix_socket_path', 'host', 'port']}
     redis_kwargs['decode_responses'] = True
+    redis_kwargs['use_unix_socket_path'] = True
```

Comment:
Where is this function getting used?

https://github.com/sonic-net/sonic-snmpagent/blob/b2aca442672713093a15b9b4054df8cb15149d73/src/sonic_ax_impl/mibs/__init__.py#L571 already uses unix_socket_path.
db = SonicV2Connector(use_unix_socket_path=True, namespace=namespace)

Reply:
I think we need to replace all `init_db` to `init_namespace_dbs`, and deprecate `init_db`.

Reply:
@SuvarnaMeenakshi I will check if line 215 can be added as part of list items in line 213, thanks

Reply:
done